overlay.d/05rhcos/usr/lib/dracut/modules.d/89ibm2810xiv-mutipath-config/ibm2810xiv-multipath-config.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
Can we change this to:

```suggestion
#!/bin/bash
set -euo pipefail
```